### PR TITLE
Try to allow scientific notation in plotting

### PR DESCRIPTION
**Pull request by kdlong**

### CODE REVIEW

**Comment by kdlong** [comment]
@davidwalter2 should I wait for your implementation of this instead?

File: wremnants/syst_tools.py [hunk 1]
```diff
@@ -132,12 +132,14 @@ def projAx(hname):
                     do_min=True)},
        "resumNPUp" : {
             "action" : lambda h: hh.syst_min_or_max_env_hist(h, projAx(hist_name), "vars", 
-                 ["c_nu-0.1-omega_nu0.5", "omega_nu0.5", "Lambda2-0.25", "Lambda20.25", "Lambda4.01", 
+                 #["c_nu-0.1-omega_nu0.5", "omega_nu0.5", "Lambda2-0.25", "Lambda20.25", "Lambda4.01",
```
Comment (davidwalter2) [inline wremnants/syst_tools.py:135]: This is not related to the change in plotting styles, is it intended and does it leave the actual results unchanged?

File: wremnants/plot_tools.py [hunk 2]
```diff
@@ -284,6 +284,8 @@ def makeStackPlotWithRatio(
             data_idx = unstacked.index("Data") 
             linestyles[data_idx] = "None"
         linestyles = np.array(linestyles, dtype=object)
+        print("Number of linestyles", len(linestyles))
+        print("Length of unstacked", len(unstacked))
```
Comment (davidwalter2) [inline wremnants/plot_tools.py:288]: If another commit is added for a different reasons the print statements could be replaced by logging

File: utilities/styles/styles.py
Comment (davidwalter2) [inline utilities/styles/styles.py:1]: For the other analyses e.g. low PU we use electrons and the \ell would be more appropriate, maybe we can find a solution that works for both. E.g. duplicating the labels with .replace("\ell", "\mu") and .replace("\ell","\mathrm{e}") and then making sure the right style is imported based on the analysis mode or so

File: scripts/plotting/postfitPlots.py [hunk 3]
```diff
@@ -190,7 +190,7 @@ def make_plot(h_data, h_inclusive, h_stack, axes, colors=None, labels=None, suff
 
     scale = max(1, np.divide(*ax1.get_figure().get_size_inches())*0.3)
 
-    fontsize = ax1.xaxis.label.get_size()
+    fontsize = ax1.xaxis.label.get_size()*0.7
```
Comment (davidwalter2) [inline scripts/plotting/postfitPlots.py:193]: (this may come from me) It would be good to avoid ad-hoc scaling of text size, this leads to inconsistent use.

File: wremnants/syst_tools.py (hunk 1 above)
Comment (kdlong) [reply wremnants/syst_tools.py:135]: It's intended, it doesn't impact the results just plotting. The variations are way too large and they look problematic in the plots, so I'm just excluding them...

File: wremnants/plot_tools.py (hunk 2 above)
Comment (kdlong) [reply wremnants/plot_tools.py:288]: I will add more commits. I am playing around here, if it's causing issues just kill the CI. Probably not worth making careful comments
